docs/src/pages/en/media-volume-range.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
no-action: weird that github didn't detect this page as moved

Reply:
heh just saw you commented on this on the other side of the fence.